boards/arm/nrf52840_pca10059/nrf52840_pca10059.dts

```diff
@@ -107,6 +126,15 @@
 	scl-pin = <31>;
 };
 
+&pwm0 {
+	status = "okay";
+	ch0-pin = <8>;
```

Comment:
This channel drives `led_1`, which is [active low](https://github.com/zephyrproject-rtos/zephyr/pull/22695/files#diff-6a7a06ff03ddb2a64d4a13dd7186ed9eL34), so you need to add `ch0-inverted` here.

Reply:
I did actually try that as that's what I thought it should be but then you never got any red as it cycled through.

Reply:
> I did actually try that as that's what I thought it should be but then you never got any red as it cycled through.

@nullr0ute Well, you will see red in the rgb_led sample when the channel 0 is inverted. You just need to wait 100 seconds (then the red LED is driven with a non-zero width pulse for the first time) for it... Or you can shorten the [sleeping period](https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/basic/rgb_led/src/main.c#L107), to see red earlier. It seems that the sample could be improved, to better present RGB LEDs in action.
When the channel is not inverted, you see red from the beginning, because then the red LED starts from being on and is (very) slowly faded off.

Reply:
@nullr0ute See also https://github.com/zephyrproject-rtos/zephyr/pull/22719. You can try with the fade_led sample after applying this patch to see that the channel for the red LED should be inverted.